store/postgres/src/entities.rs

```diff
@@ -905,7 +905,38 @@ impl Table {
                    $2 as subgraph,
                    $3 as entity,
                    $4 as entity_id,
-                   (select data from {}.entities where entity = $3 and id = $4) as data_before,
+                   (select data
+                      from {}.entities
+                     where subgraph = $2
+                       and entity = $3
+                       and id = $4) as data_before,
+                   $5 as reversion,
+                   $6 as op_id",
+                schema, schema,
```

Comment:
We know here that `schema == "public"` and could write that into the query, though it's a minor improvement.